fastcov.py

```diff
@@ -282,6 +282,14 @@ def exclMarkerWorker(fastcov_sources, chunk, exclude_branches_sw, include_branch
             for test_name in fastcov_sources[source]:
                 fastcov_data = fastcov_sources[source][test_name]
 
+                # Build line to function dict so can quickly delete by line number
+                line_to_func = {}
+                for f in fastcov_data["functions"].keys():
+                    l = fastcov_data["functions"][f]["start_line"]
+                    if l not in line_to_func:
+                        line_to_func[l] = {}
+                    line_to_func[l][f] = f
```

Comment:
```suggestion
                        line_to_func[l] = set()
                    line_to_func[l].add(f)
```
I recommend using a `set()` here instead of `dict()` since the the value is a don't care